packages/specification/schemas/resources/OrganisationDispensing.yaml

```diff
@@ -0,0 +1,63 @@
+type: object
+description: Details of the dispensing organisation for a prescription.
```

Comment:
Details of the pharmacy that a prescription is going to be dispensed by.

Reply:
Changed as suggested.